src/main.js

```diff
@@ -204,8 +200,8 @@ function addGrid() {
   var posterHTML = '';
 
   for (var i = 0; i<savedPosters.length; i++) {
-    posterHTML += `<article class="mini-poster">
-    <img src=${savedPosters[i].imageURL} />
+    posterHTML += `<article class="mini-poster" ondblclick="remove(this)">
```

Comment:
Great use of this code and it was particularly helpful when you used the console to show exactly what is being replaced in the live code.